labs/aifaq.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: default
+title: AI FAQ for Hyperledger Ecosystem
+[//]:# (parent: Labs)
+---
+
+# aifaq
+
+AI FAQ for Hyperledger Ecosystem
+
+# Short Description
+
+AI FAQ provides a ChatBot that replies to the questions on Hyperledger Ecosystem: that is a Generative Pre-Trained Model based on Transformer technology. The system has a modern QA architecture based on Retriever-Reader pipeline. The Retriever component has the purpose of select the most relevant documents, based on similarity of the question. While the Reader component returns the answer to the user, with a snippet of text extracted from the documentation. The final version will use the most sophisticated technologies (RAG, QLoRA, DPO, ...).
+
+# Scope of Lab
+
+The scope of the lab is to support the Hyperledger users (users, developer, etc.) to their work, avoiding to wade through oceans of documents to find information they are looking for. Large Language Models have yielded remarkable results, either pay and open source tools. Today we can implement a conversational AI tool which replies to questions related to specific context.
+
+# Initial Committers
+
+https://github.com/gcapuzzi
+
+# Sponsor
+
+- Tracy A. Kuhrt - https://github.com/tkuhrt
```

Comment:
We have not talked about me being the sponsor for this lab. Should your mentor be the sponsor?

Reply:
Sorry, I apologize she cannot

Reply:
I asked again: she can and I changed labs/aifaq.md file